pkg/webhookadmission/webhook.go

```diff
@@ -130,16 +131,23 @@ func (c *Controller) processMutateRequest(w http.ResponseWriter, req *http.Reque
 			Allowed: true,
 		}
 	} else {
-		// pod object does not have name and namespace populated, so we pass them separately. Also,
-		// if the pod is using generateName, arReq.Name is empty.
-		patches, err = c.Driver.GetPodPatches(arReq.Namespace, &pod)
-		if err != nil {
-			log.Errorf("Failed to get pod patches for pod %s/%s: %v", arReq.Namespace, resourceName, err)
-			c.Recorder.Event(webhookConfig, v1.EventTypeWarning, "could not get pod patches", err.Error())
-			http.Error(w, "Could not get pod patches", http.StatusInternalServerError)
-			return
+		// `kubevirt-skip-preload-statfs=true` stork argument can be configured
+		// by the user as per their need through which stork can get to know if to
+		// actually create the configmap and mount the ld.so.preload and px_statfs.so
+		// shared lib onto the virt-launcher container.
+		if c.KubevirtSkipPreloadStatFS {
```

Comment:
As I mentioned in my previous comment, we don't want to skip all pod patches. I think we need to put this check here:
https://github.com/libopenstorage/stork/blob/c81f52d5289ce141c686063c9a8e8b21a10332d4/drivers/volume/portworx/portworx.go#L4348